ami.json

```diff
@@ -41,7 +41,7 @@
         "sudo yum -y clean all",
         "sudo yum -y upgrade",
         "sudo yum search java",
-        "sudo yum install -y java-1.8.0-openjdk-devel",
+        "sudo yum install -y java-17-openjdk-devel",
```

Comment:
Does the AMI build?

Reply:
Don't have an AWS account, but I checked if this package was valid

Reply:
and it was

Reply:
https://access.redhat.com/documentation/en-us/openjdk/17/html-single/installing_and_using_openjdk_17_on_rhel/index

Over here these guys use the same command

Reply:
Amazon Linux 2 uses different package providers than Red Hat / CentOS. This likely doesn't work. I suspect only jdk 17 corretto is available on AL2.. but I'll fix that when the time comes...